apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -39,6 +40,7 @@ const CurriculumCatalog = ({
   isSignedOut,
   isTeacher,
   curriculaTaught,
+  forceTranslated,
```

Comment:
Added a property to not allow switching to a filter by the current locale (instead ALWAYS filter by that locale). Required for rendering within regions like Farsi.